swagger.yaml

```diff
@@ -155,5 +317,116 @@ components:
           format: date
         assignedPersons:
           type: array
-          items: 
-            type: integer
\ No newline at end of file
+          items:
+            type: integer
+
+    Leads:
+      type: object
+      description: Lead data from Pipedrive API
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        usedTech:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        add_time:
+          type: string
+          format: date-time
+        update_time:
+          type: string
+          format: date-time
+        next_activity_date:
+          type: string
+          format: date-time
+        label_ids:
+          type: array
+          items:
+            type: string
+    Deals:
+      type: object
+      description: Deal data from Pipedrive API
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        usedTech:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        value:
+          type: number
+        currency:
+          type: string
+        add_time:
+          type: string
+          format: date-time
+        update_time:
+          type: string
+          format: date-time
+        next_activity_date:
+          type: string
+          format: date-time
+        status:
+          type: string
+        next_activity_subject:
+          type: string
+        next_activity_note:
+          type: string
+
+    LeadOrDeal:
```

Comment:
When this endpoint is split as I described you should be able to remove this schema and use the existing deal or lead schema